css/main.css

```diff
@@ -0,0 +1,264 @@
+/* Start Fonts */
+@font-face {
+  font-family: cocogoose;
+  src: url(../fonts/cocogoose/Cocogoose-Pro-Thin-trial.ttf);
+  font-weight: 100;
+}
+
+@font-face {
+  font-family: cocogoose;
+  src: url(../fonts/cocogoose/Cocogoose-Pro-Ultralight-trial.ttf);
+  font-weight: 200;
+}
+
+@font-face {
+  font-family: cocogoose;
+  src: url(../fonts/cocogoose/Cocogoose-Pro-Light-trial.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: cocogoose;
+  src: url(../fonts/cocogoose/Cocogoose-Pro-Semilight-trial.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: cocogoose;
+  src: url(../fonts/cocogoose/Cocogoose-Pro-trial.ttf);
+  font-weight: 600;
+}
+
+/* Start Global Rules */
+* {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body,
+html {
+  overflow-x: hidden;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-family: cocogoose, sans-serif;
+}
+
+a {
+  text-decoration: none;
+  color: #d3d3d3;
+  font-family: 'Lato', sans-serif;
+}
+
+a:hover {
+  color: #d3d3d3;
+}
+
+.me-70 {
+  margin-right: 70px;
+}
+
+li,
+p,
+span {
+  font-family: 'Lato', sans-serif;
+}
+
+/*
+  color: #272a31;
+  color: #ec5242;
+  color: #d3d3d3;
+  font-family: 'Lato', sans-serif;
+  font-family: cocogoose, sans-serif;
+*/
+
+/* Start Navbar */
```

Comment:
**Keep your project clean and professional**
- Remove unused code and commented lines. Your code should be explain by itself about what it do.